### DISCUSSION TITLE
v4.9.0-rc.2

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5824] [Announcements]
Code v1.73.1

This is the second pre-release for v4.9.0. 

To install via the install script, pass the `--edge` flag. 

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.9.0-rc.2'>v4.9.0-rc.2</a>.</em>

**Comment by jsjoeio** [comment]
Linux users: could you check to make sure the Terminal works? I tested on macOS and didn't see any issues.

cc @benz0li

**Reply by benz0li** [reply]
The issue with `/opt/code-server/lib/vscode/product.json`[^1] remains. It should have permissions `644` not `600`.
ℹ️ Tested with code-server [v4.9.0-rc.2](https://github.com/coder/code-server/releases/tag/v4.9.0-rc.2) (`code-server-4.9.0-rc.2-linux-amd64.tar.gz`)

Cross reference: https://github.com/coder/code-server/pull/5769#issuecomment-1311337491

[^1]: `/opt/code-server`: Installation path

**Reply by benz0li** [reply]
code-server [v4.9.0-rc.2](https://github.com/coder/code-server/releases/tag/v4.9.0-rc.2) (`code-server-4.9.0-rc.2-linux-amd64.tar.gz`; fixed version with `CS_DISABLE_GETTING_STARTED_OVERRIDE=1` set) is deployed at https://coder.jupyter.b-data.ch; Image `R (base:test-devtools)`.

Functionality [modified by patches] tested and found to work:

* base-path
* cli-window-open
* local-storage
* marketplace
* proxy-url
* service-worker
* webview

Jupyter Notebooks also work fine:

*  `ms-toolsai.jupyter@2022.10.110`
*  `ms-python.python@2022.18.2`

**Reply by jsjoeio** [reply]
> The issue with /opt/code-server/lib/vscode/product.json[1](https://github.com/coder/code-server/discussions/5824#user-content-fn-1-f914d5cc7fae2578583e717f32d185f7) remains. It should have permissions 644 not 600.

Interesting 🤔 I wonder how that happened. I will update in https://github.com/coder/code-server/pull/5772 and then ping you to help test.

**Reply by jsjoeio** [reply]
This is normally something I'd ping @code-asher for help with but he is out until next week. I _think_ this should do it: https://github.com/coder/code-server/pull/5772/commits/fc4d2b532f41277d6b93243cd6bd2ba2434f80d6

I ran `build:vscode` locally and checked permissions with: `stat -f%p product.json` and got `100755` so I think that should fix it? I'm not certain though. 

CI is building and when it's done we'll have the npm artifact. 

If you need me to do another release-candidate for testing, let me know.

**Reply by jsjoeio** [reply]
I guess it's probably best to do a release candidate then others can test too. Plus it tests in the same way (linux-amd package) vs. the npm artifact in CI. I'll do that now.
